contracts/src/bridge/ISequencerInbox.sol

```diff
@@ -154,6 +156,13 @@ interface ISequencerInbox is IDelayedMessageProvider {
      */
     function invalidateKeysetHash(bytes32 ksHash) external;
 
+    /**
+     * @notice Updates whether an address is authorized to be a sequencer at the sequencer inbox
+     * @param addr the address
+     * @param isSequencer_ if the specified address should be authorized as a sequencer
```

Comment:
I think we should add a @dev here that this is not validated onchain right now.

Reply:
I added a dev comment describing briefly where the IsSeqeuncer information is used. Please let me know if that's what you had in mind